src/app/core/components/header/header.component.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../styles/colors";
+
+.mat-sidenav-container{
+    height: 100%;
+    background-color: $dark;
+
+    .mat-sidenav {
+        padding: 20px;
+        background-color: $grey;
+        width: 20%;
+    }
+
+    .router-link {
+        background-color: blanchedalmond;
+        color: blue;
+    }
+
+    .mat-toolbar{ 
+        color: white;
+        height: 80px;
+        background-color: $grey;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+
+        .sidebar {
+            display: flex;
+            align-items: center;
+
+            p {
+                margin: 0;
+            }
+        }
+
+        .nav {
+
```

Comment:
This empty line is redundant, you can remove it

Reply:
fixed